rts/Game/UI/SelectionKeyHandler.cpp

```diff
@@ -367,6 +368,8 @@ void CSelectionKeyHandler::DoSelection(std::string selectString)
 
 	ReadDelimiter(selectString);
 
+	idMatchesSet.clear();
```

Comment:
Why a static var and not just a regular var on the stack declared here?

Reply:
Good point, the closure is unnecessary. I think previous iteration used the scope.